tests/EvmAcceptanceTests/test/rpc/eth_getBlockByNumber.js

```diff
@@ -127,49 +127,72 @@ describe("Calling " + METHOD, function () {
     });
   });
 
-  it("should return get full transactions objects by 'earliest' tag", async function () {
-    await helper.callEthMethod(METHOD, 2, ["earliest", true], (result, status) => {
-      hre.logDebug(result);
+  describe("When on network", function () {
+    let blockNumber = 0x0;
+    before(async function () {
+      if (hre.network.name == "isolated_server") {
+        this.skip(); // isolated server does not returns block '0' back, but a null object.
```

Comment:
if this is a bug for isolated server, we can have a `FIXME` comment here with the related issue number included in.

Reply:
i have a ticket for it